code/obj/item/gun/ammo.dm

```diff
@@ -675,6 +675,35 @@
 	icon_empty = "12-0"
 	sound_load = 'sound/weapons/gunload_heavy.ogg'
 
+/obj/item/ammo/bullets/pipeshot
+	sname = "pipeshot"
+	name = "pipeshot"
+	desc = "A parent item! If you see this contact a coder."
+	ammo_type = new/datum/projectile/special/spreader/buckshot_burst
+	icon_state = "makeshiftempty"
+	amount_left = 4.0
+	max_amount = 4.0
+	ammo_cat = AMMO_SHOTGUN_SCRAP
+	delete_on_reload = TRUE
+	icon_dynamic = FALSE
+	icon_empty = "makeshiftempty"
```

Comment:
If `delete_on_reload = TRUE` is this `icon_empty` ever used?

Reply:
nope, will remove